### DISCUSSION TITLE
[Feature Request] 关于在线音乐文件播放

### DISCUSSION THREAD

**Discussion by ghost** [spencerwooo/onedrive-vercel-index#182] [Ideas]
## 需求
希望可以设定一个目录为音乐目录，在打开该目录的“根目录”页面时，网页会扫描该目录中的一级目录的音乐文件, 形成一个音乐列表，使得能够在一个页面上播放该目录中的所有音乐文件。
## 附加描述
参考网易音乐平台网页版

![music](https://user-images.githubusercontent.com/68503296/143252258-56b88e2f-7888-4933-87dd-162fd17599f9.png)

**Comment by Mahoo12138** [comment]
这个不如直接使用类似海贝音乐的APP绑定Onedrive了
